### PR TITLE
feat: add `pypi` package `torch-frft` recipe

### PR DESCRIPTION
**Pull request by tunakasif**
Add a recipe for the `torch-frft` package, which is already available on PyPI.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tunakasif** [comment]
Hi @conda-forge/help-python, this is ready for review. As the recipe is `noarch: python`, the Windows error does not matter (no PyTorch builds for Win).

**Comment by tunakasif** [comment]
@conda-forge/help-python, ready for review!

Please note this is a UNIX-only package - I added `__unix` to the run requirements of the package to ensure this. Thanks for considering 😄

**Comment by tunakasif** [comment]
@conda-forge-admin, please ping conda-forge/help-python, ready for review!

File: recipes/torch-frft/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  host:
+    - python >=3.10,<3.12
+    - poetry-core
+    - pip
+  run:
+    - python >=3.10,<3.12
+    - numpy >=1.24.3,<2.0.0
+    - scipy >=1.10.1,<2.0.0
+    - pytorch >=2.0.0,<2.0.1
+    - __unix
```
Comment (ocefpaf) [inline recipes/torch-frft/meta.yaml:27]: Id the reason for this is the lack of pytorch on Windows, you don't need it. The package won't be installable on Windows as anyway and, if pytorch is ever made available there, this package would be ready. With the `__unix` selector it won't.

```suggestion
```

Comment (tunakasif) [reply recipes/torch-frft/meta.yaml]: It is exactly like what you've described, thank you for the suggestion 👍🏻

**Comment by tunakasif** [comment]
@conda-forge-admin, please ping conda-forge/help-python, ready for review!

**Comment by tunakasif** [comment]
@ocefpaf Thank you for your help 🎉

**Pull request merged**